controller/payload/payload_scheduler.py

```diff
@@ -694,3 +697,99 @@ def add_information_source_statistics_exclusion(
         if idx % 2 == 0
     ]
     general.add_all(exclusions, with_commit=True)
+
+
+def prepare_sample_records_doc_bin(
+    project_id: str, information_source_id: str
+) -> Tuple[str, List[str]]:
+    sample_records = record.get_attribute_calculation_sample_records(project_id)
+
+    sample_records_doc_bin = get_doc_bin_table_to_json(
+        project_id=project_id,
+        missing_columns="",
```

Comment:
if i'm not toally mistaken this will result in some errors when we access the data of a collumn outside the docbins (e.g. running_id) since the missing_columns aren't provided